wallet-api/src/WalletAPI/UtxoStore.hs

```diff
@@ -0,0 +1,47 @@
+module WalletAPI.UtxoStore where
+
+import           RIO
+import qualified Data.Set               as Set
+import qualified Data.Map               as Map
+
+import Ledger
+import CardanoTx.Models
+
+type Store = Map.Map TxOutRef FullTxOut
+
+data UtxoStore f = UtxoStore
+  { putUtxos     :: Set.Set FullTxOut -> f ()
+  , getUtxos     :: f (Set.Set FullTxOut)
```

Comment:
Should we add API with a batch selection? For example, we don't want to get all utxo's but only 10(we are sure that it's enough for us, if not, just take the other 10)

Reply:
In which case are we sure 10 is enough?

Reply:
Or any other number